components/Wrapper.js

```diff
@@ -0,0 +1,81 @@
+import Head from 'next/head';
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+
+import Footer from './Footer';
+import Navigation from './Navigation';
+
+import { formatDate } from '../lib/date';
+import { initGA, logPageView } from '../lib/analytics';
+import { styles } from '../constants/styles';
+
+export default class Wrapper extends Component {
+  componentDidMount() {
+    if (!window.GA_INITIALIZED) {
+      initGA();
+      window.GA_INITIALIZED = true;
+    }
+    logPageView();
+  }
+  renderDate() {
+    if (this.props.data.date) {
+      return (
+        <time className='mid-gray ttu f6 f5-ns'>
+          {formatDate(this.props.data.date)}
+        </time>
+      );
+    }
+    return null;
+  }
+
+  renderHeader() {
+    if (this.props.data.header) {
+      return (
+        <header className={`${styles.wrapper}`}>
+          <div className='measure-wide mt5 mb4'>
+            <h2 className='f3 f2-ns mv0 near-black mb0'>
+              {this.props.data.header}
+            </h2>
+            {this.renderDate()}
+          </div>
+        </header>
+      );
+    }
+    return null;
+  }
+
+  render() {
+    const { data, className, children: content } = this.props;
+
+    const title = data.title ? `${data.title}` : 'Clouty';
```

Comment:
good use case for [nullish coalescing](https://github.com/tc39/proposal-nullish-coalescing)
```suggestion
    const title = data.title ?? 'Clouty';
```